src/components/Settings/Settings.tsx

```diff
@@ -1,8 +1,13 @@
 import { Component, h } from "preact";
 import { SettingsDevices } from "./Settings-devices";
-import SettingsIcon from "./settings.svg";
 import { AudioIndicator } from "../AudioIndicator/AudioIndicator";
 import { Translator } from "../Translator/Translator";
+import VideoIcon from "./assets/video.svg";
+import AudioIcon from "./assets/audio.svg";
+import ScreenIcon from "./assets/screen.svg";
+import NoVideoIcon from "./assets/noVideo.svg";
+import NoAudioIcon from "./assets/noAudio.svg";
+import NoScreenIcon from "./assets/noScreen.svg";
```

Comment:
when importing icons, which are pretty much similar. I tend to import like:
`import { icon1, icon2, icon3 } from '../path/to/icons/folder'`

this will keep the code more organized